wen-gong/week7/homework day1/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>AJAX Intro</title>
+    <script src="js/jquery.js"></script>
+    <script src="js/main.js"></script>
+
+</head>
+<body>
+
+    <h1>Movie!</h1>
+
+    <input type="text" id="movieQuery" placeholder="Enter a movie">
+    <button id="submitButton">Lookup Number Trivia</button>
```

Comment:
If you use or copy code from somewhere else, please make sure to update the names.